.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,19 @@
+### Describe your changes
+
+### Issue ticket number and link if any
+
+### Checklist for finalizing the PR
+
+- [ ] I have performed a [self-review](https://f3d.app/doc/dev/CODING_STYLE.html) of my code
+- [ ] I have added [tests](https://f3d.app/doc/dev/TESTING.html) for new features and bugfixes
+- [ ] I have added [documentation](https://f3d.app/doc/dev/TESTING.html) for new features
+- [ ] If it is a modifying the libf3d API, I have updated bindings
+- [ ] If it is a modifying the default versions, I have updated timestamp
+
+### Continuous integration
+
+Please check the checkbox of the CI you want to run, then push again on your branch.
+
+- [ ] Style checks
+- [ ] Fast CI
+- [ ] Full CI
```

Comment:
Should we add more checkboxes? I'm thinking of `android`/`wasm` and maybe `macos`/`linux`/`windows` too?

Reply:
In order to avoid running full CI when working on `android` feature for example ?

Reply:
yes that's the idea